static/scss/partials/_main.scss

```diff
@@ -115,6 +115,15 @@ a {
             font-size: 18px;
         }
     }
+    a.card-link {
+        color: inherit;
+        font-weight: inherit;
+        text-decoration: inherit;
+
```

Comment:
After testing it I've realized the `a` element does not fill the whole card if the text is not big enough.
Please add `height: 100%;` here.